### PR TITLE
Implement Dockerfile for local testing

### PR DESCRIPTION
**Pull request by p365labs**
Hi all I've tried to implement a Dockerfile for testing on local environment.
This will allow idempotent test on differents workstations with different OS
and fixing library versions (php & pecl & so on)

what do u think?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,25 +31,38 @@ Thank you!
...
 
 ```bash
-SYMFONY_VERSION=3.4.1 ./runTests.sh --filter testCustomFileNameProperty
+make test TEST="Tests/Util/FilenameUtilsTest.php"
+```
+
+will allow to tests single Test Classes.
+
+There are 3 environments available: PHP 7.2, 7.3 and 7.4-rc.
+Default environment is *PHP 7.2* if you want to execute it against
```
Comment (garak) [inline CONTRIBUTING.md]: Is there a reason for 7.2 being default? I guess that 7.3 would be a better choice, being the latest stable version.

File: composer.json [hunk 2]
```diff
@@ -47,6 +47,7 @@
         "symfony/framework-bundle": "^3.4.23 || ^4.2.4",
         "symfony/phpunit-bridge": "^4.3",
         "symfony/security-csrf": "^3.4 || ^4.0",
+        "symfony/symfony": "~3.4.0",
```
Comment (garak) [inline composer.json]: Why this add to requirements?

File: CONTRIBUTING.md (hunk 1 above)
Comment (p365labs) [reply CONTRIBUTING.md]: no absolutely no a real reason. u're right! going to update it.

File: composer.json (hunk 2 above)
Comment (p365labs) [reply composer.json]: my mistake... I forgot to remove it.

File: .gitattributes [hunk 3]
```diff
@@ -11,3 +11,5 @@
 /Tests              export-ignore
 /Resources/meta     export-ignore
 /Resources/doc      export-ignore
+/docker             export-ignore
+/.Makefile          export-ignore
```
Comment (garak) [inline .gitattributes]: There's a typo here: it should be `/Makefile`, not `/.Makefile`

Comment (p365labs) [reply .gitattributes]: ahahahah my bad :( sorry @garak

**Comment by garak** [comment]
Thank you @p365labs

**Pull request merged**